src/ComparedQueryable/QueryableExtensions.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections.Generic;
+using System.Linq;
+using NaturalSort;
+
+namespace ComparedQueryable
+{
+    public static class QueryableExtensions
+    {
+        /// <summary>
+        /// Converts an <see cref="IEnumerable{T}"/> into an <see cref="IQueryable{T}"/> the <see cref="string"/> 
+        /// properites of which can be naturally sorted.
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <param name="collection"></param>
+        /// <returns></returns>
+        public static IQueryable<T> AsNaturalQueryable<T>(this IEnumerable<T> collection)

```

Comment:
This is a practical application of the generic extension method below that sorts strings naturally (the point of this PR).